### PR TITLE
feat: Generate and display random cron expression and its human readable form

### LINKED ISSUE #2: generate random cron and language expression
On page load,
we need to generate the following
1. `1 * * * *` - the cron value
2. The human readable language equivalent (https://github.com/bradymholt/cRonstrue)

And display them side-by-side.

Scope your PR to only this, no other additional logic/styling/css.

### PR DESCRIPTION
**Pull request by glokta1**
Generates very basic cron expressions alongside its human readable form. 

`*` 70% of the time for each placeholder, random (allowed) value otherwise.

Resolves #2

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
-        Click on the Vite and React logos to learn more
-      </p>
+      <h1>crontab.training</h1>
+      <p>{cronExpression}</p>
+      <p>{cronstrue.toString(cronExpression)}</p>
     </>
   )
 }
 
+function getRandomIntInclusive(min: number, max: number) {
```
Comment (arvindpunk) [inline src/App.tsx:33]: Can we use [crypto.randomInt](https://nodejs.org/api/crypto.html#cryptorandomintmin-max-callback) instead?

Also, move the generator function to a different file, `generator.ts`.

**Review by arvindpunk — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (glokta1) [reply src/App.tsx:33]: > Can we use [crypto.randomInt](https://nodejs.org/api/crypto.html#cryptorandomintmin-max-callback) instead?
Getting the following error: `Uncaught Error: Module "crypto" has been externalized for browser compatibility. Cannot access "crypto.randomInt" in client code.`

Relevant documentation: https://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility

How should we proceed?

Comment (arvindpunk) [reply src/App.tsx:33]: Gotcha, this is a nodejs only package. Let's stick to your implementation for now. The end goal is to move generation and validation to a backend server anyway.

**Review by arvindpunk — APPROVED** [review]
LGTM.

**Pull request merged**